app/controllers/EPayeStartController.scala

```diff
@@ -46,29 +48,38 @@ class EPayeStartController @Inject() (
     Ok(ePayeLandingPage())
   }
 
-  val ePayeLandingSubmit: Action[AnyContent] = as.default { implicit request =>
+  val ePayeLandingSubmit: Action[AnyContent] = as.default { _ =>
     Redirect(routes.EPayeStartController.ePayeStart())
   }
 
   val ePayeStart: Action[AnyContent] = as.default.async { implicit request =>
     request.session.data.get("JourneyId") match {
       case Some(_: String) => for {
-        data <- eligibilityDataService.data("AOR", TaxRegime.Epaye, ttp.DefaultTaxId, true)
+        data <- eligibilityDataService.data(
+          idType = "AOR",
+          regime = TaxRegime.Epaye,
+          id = ttp.DefaultTaxId,
+          showFinancials = true)
       } yield routeResponse(data)
       case _ => throw new IllegalStateException("missing journey")
     }
   }
 
-  val ePayeStartSubmit: Action[AnyContent] = as.default { implicit request =>
+  val ePayeStartSubmit: Action[AnyContent] = as.default { _ =>
     Redirect(routes.UpfrontPaymentController.upfrontPayment())
   }
 
   def routeResponse(data: EligibilityData)(implicit R: Request[_]): Result = {
     if (data.hasRejections) {
-      val errorMessage = data.rejections.map(_.entryName).mkString(", ")
-      Ok(errorTemplate("Error Page", "TTP Rejections", errorMessage)(implicitly[Request[_]], R.messages))
+      val leadingContentToShow: Seq[Message] = data.rejections.head match {
```

Comment:
this method of using Messages alone might not be best for when we have more complex HTML structures with links, just commenting really as it obviously works for this use case, but we maybe could move to partials being included e.g DebtIsTooLarge.scala.html in later tickets if it proves a better way of handling the variable content

Reply:
Yeah I was thinking this, will refactor when the time comes